tests/src/common/WskTestHelpers.scala

```diff
@@ -118,3 +118,22 @@ trait WskTestHelpers extends Matchers {
         }
     }
 }
+
+object JsObjectExtensions {
```

Comment:
@rabbah I suspect there might be a more appropriate file to put this in. Thoughts?